components/cards/ApplicationStageTimelineCard.tsx

```diff
@@ -5,13 +5,14 @@ import { isValidDate } from '../../utils/date/validations';
 
 interface Props {
   applicationStage: ApplicationStageApplicationData;
+  onClick?: () => void;
 }
 
-function ApplicationStageTimelineCard({ applicationStage }: Props) {
+function ApplicationStageTimelineCard({ applicationStage, onClick }: Props) {
   const date = isValidDate(applicationStage.date) ? new Date(applicationStage.date) : undefined;
 
   return (
-    <div className="shadow-md rounded-lg">
+    <div className="shadow-md rounded-lg" onClick={onClick}>
```

Comment:
Should we make the cursor a pointer and perhaps darken the background when the user is hovering over the div? To make it clear that it is clickable (for Web users).